src/levenberg_marquardt.jl

```diff
@@ -0,0 +1,170 @@
+immutable LevenbergMarquardt <: Optim.Optimizer end
```

Comment:
If this is no longer in the `Optim` package, what are we gaining by subclasses `Optim.Optimizer`?